docker/Dockerfile.agent.multiarch.rootless

```diff
@@ -1,12 +1,15 @@
 FROM --platform=$BUILDPLATFORM docker.io/golang:1.23 AS build
 
+RUN adduser -u 1000 -g 1000 woodpecker && \
```

Comment:
Sam here. We dont need to chain but at least position the RUN commands together?